internal/command/views/test.go

```diff
@@ -729,10 +729,7 @@ func (t *TestJSON) TFCStatusUpdate(status tfe.TestRunStatus, elapsed time.Durati
 }
 
 func (t *TestJSON) TFCRetryHook(attemptNum int, resp *http.Response) {
-	t.view.log.Error(
```

Comment:
@liamcervante I’m not sure what to do here. Initially, we wrapped the retryLogHook in this error call, the test still passes. What do you recommend we do here?